apps/settings/style/simcard.css

```diff
@@ -139,11 +139,7 @@
   color: #6d7374;
 }
 
-/*
- * Right to Left layout
- */
-
+/* RTL View */
```

Comment:
Same here. Unless I’m missing a point, I don’t see why we should break the consistency — even for separation comments.